src/foam/nanos/cron/CronSchedule.js

```diff
@@ -93,6 +93,18 @@ foam.CLASS({
       ],
       documentation: 'comma seperated hours',
     },
+    {
+      documentation: 'deprecated, replaced by monthsOfYear',
+      class: 'Int',
+      name: 'month',
+      transient: true,
+      hidden: true,
+      javaSetter: `
+      if ( val > 0 && ! monthsOfYearIsSet_ ) {
```

Comment:
Do we want to append the value to the array if it is already defined and has value in it? Same question for all the java setters supporting deprecated properties.

Reply:
No.  If there is already a value then the new property is being used.

Reply:
if month is -1, shouldn't we add every month to monthsOfYear?

Reply:
Yes, since -1 was wildcard. Need to update all.